### PR TITLE
Move cmake to the build section.

### PR DESCRIPTION
**Pull request by vyasr**
As a CLI tool CMake belongs in the build section and shouldn't need to be present in the host requirements.

### CODE REVIEW

**Review by msadang — APPROVED** [review]
Version changes and cmake build look good.

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+# raft 22.10.00 (Date TBD)
```
Comment (raydouglass) [inline CHANGELOG.md:1]: Looks like you branched from `branch-22.10` but are targeting `branch-22.08`. Can you fix this?

**Review by cjnolet — APPROVED** [review]

**Comment by cjnolet** [comment]
Looks like an intermittent failing ANN ivf flat gtest.

**Comment by cjnolet** [comment]
rerun tests

**Comment by cjnolet** [comment]
@gpucibot merge

**Pull request merged**